.github/scripts/verifyActions.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Re-compiles all Github Actions and verifies that there is no diff,
+# because that would indicate that the PR author forgot to run `npm run gh-actions-build`
+# and commit the re-bundled the javascript sources.
+
+declare -r GREEN='\033[0;32m'
+declare -r RED='\033[0;31m'
+declare -r NC='\033[0m'
+
+if [[ $(git diff --exit-code) -eq 0 ]]; then
```

Comment:
Should we run the `ncc` command in this script? As it's a bit more contained?

Reply:
Yeah, good suggestion. I moved the rebuild step into this script (although it's still using `npm run gh-actions-build`)